hoot-core-test/src/test/cpp/hoot/core/algorithms/AlphaShapeTest.cpp

```diff
@@ -33,13 +33,11 @@
 
 // Hoot
 #include <hoot/core/OsmMap.h>
+#include <hoot/core/TestUtils.h>
 #include <hoot/core/algorithms/AlphaShape.h>
 #include <hoot/core/io/OsmXmlWriter.h>
 #include <hoot/core/util/Log.h>
 
-// Qt
-#include <QDir>
-
```

Comment:
Quite a few files include `QDir` that don't need it anymore since they call `TestUtils::mkpath` to create directories instead of using `QDir` directly.